test-cases/test-cases.go

```diff
@@ -135,32 +135,55 @@ const htmlTemplate = `
 </html>
 `
 
+var newLineRegexp = regexp.MustCompile(`(\r\n)|[\n\r]`)
+
+type formatType string
+
+const (
+	formatText       formatType = "text"
+	formatColourText            = "colour-text"
```

Comment:
Test-cases has always been a UK English application.  This commit is replacing a --colour option with a --format=colour-text, so it's just maintaining the status quo.